lib/manager/helm-values/update.ts

```diff
@@ -0,0 +1,118 @@
+import YAWN from 'yawn-yaml/cjs';
+import { logger } from '../../logger';
+import { Upgrade } from '../common';
+import { matchesHelmValuesDockerHeuristic } from './util';
+
+function shouldUpdate(
+  parentKey: string,
+  data: any,
```

Comment:
```suggestion
  data: unknown | DockerImage,
```

For `DockerImage` see below